src/EditorFeatures/Core/Implementation/ExtractMethod/AbstractExtractMethodCommandHandler.cs

```diff
@@ -220,7 +220,7 @@ private bool TryNotifyFailureToUser(Document document, ExtractMethodResult resul
             {
                 options = options.WithChangedOption(ExtractMethodOptions.DontPutOutOrRefOnStruct, document.Project.Language, true);
                 var newResult = ExtractMethodService.ExtractMethodAsync(
-                    document, spans.Single().Span.ToTextSpan(), options, cancellationToken).WaitAndGetResult(cancellationToken);
+                    document: document, textSpan: spans.Single().Span.ToTextSpan(), localFunction: false, options: options, cancellationToken).WaitAndGetResult(cancellationToken);
```

Comment:
no need for *most* of these named parameters.  you can just have the single one for `localFunction: false`.